templates/ghostty-base16.mustache

```diff
@@ -0,0 +1,35 @@
+# vim: ft=ghostty
+# {{ scheme-name }} theme for Ghostty
+# Scheme Author: {{ scheme-author }}
+# Scheme System: {{ scheme-system }}
+# Template Author: Bez Hermoso
```

Comment:
```suggestion
# Template Author: Tinted Terminal (https://github.com/tinted-theming/tinted-terminal)
```

Reply:
We've set the template authors to be tinted-theming/tinted-terminal so we can have a group maintain and not have templates belong to a single person.